### PR TITLE
Сенина - Лабораторная работа #2

### PR DESCRIPTION
**Pull request by AnastasiaaSenina**

### CODE REVIEW

**Comment by AnastasiaaSenina** [comment]
Please review @JuliaKamelina  @UsovaMA

File: code/my_set/viewmodel/viewmodel.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    def add(self, data, name_set):
+        try:
+            if name_set == 'A':
+                processed_data = eval(data)
+                self.set_A.add(processed_data)
+            else:
+                processed_data = eval(data)
+                self.set_B.add(processed_data)
+        except Exception as exp:
+            self.error_msg = "Error: {}". format(exp)
```
Comment (JuliaKamelina) [inline code/my_set/viewmodel/viewmodel.py:20]: Now, when value of incorrect type entered (e. g. `'a'`) the message shown is `Error: name 'a' is not defined`. Probably it is better not to show the user internal error messages, but notify that the entered value is of incorrect type.

File: code/my_set/viewmodel/viewmodel.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    def delete(self, data, name_set):
+        try:
+            if name_set == 'A':
+                processed_data = eval(data)
+                self.set_A.delete(processed_data)
+            else:
+                processed_data = eval(data)
+                self.set_B.delete(processed_data)
+        except Exception as exp:
+            self.error_msg = "Error: {}". format(exp)
```
Comment (JuliaKamelina) [inline code/my_set/viewmodel/viewmodel.py:31]: Same

File: code/my_set/viewmodel/viewmodel.py (hunk 1 above)
Comment (AnastasiaaSenina) [reply code/my_set/viewmodel/viewmodel.py]: Done.

**Review by JuliaKamelina — APPROVED** [review]
👍

**Review by UsovaMA — APPROVED** [review]
:+1:

**Pull request merged**